frappe/integrations/doctype/webhook/__init__.py

```diff
@@ -21,7 +21,9 @@ def run_webhooks(doc, method):
 		if webhooks is None:
 			# query webhooks
 			webhooks_list = frappe.get_all('Webhook',
-				fields=["name", "`condition`", "webhook_docevent", "webhook_doctype"])
+						fields=["name", "`condition`", "webhook_docevent", "webhook_doctype"], 
+						filters={"enabled": "1"}
```

Comment:
Keep this as:

```suggestion
						filters={"enabled": True}
```

The string value is only for the database.

Reply:
I personally thought so, but I had come across instances in the codebase where "1" is used (ERPNext). I think True is better, will change in next commit to this PR.